client/store/plugins/vuex-persist.js

```diff
@@ -0,0 +1,11 @@
+import VuexPersistence from 'vuex-persist';
+
+const MODULES_TO_PERSIST = ['auth'];
+
+export default new VuexPersistence({
+  // The key to store the state on in the storage provider.
+  key: process.env.PERSIST_STORAGE_KEY,
+  modules: MODULES_TO_PERSIST,
```

Comment:
What did we gain with extracting this 🤔

Reply:
I have extracted it because other modules can be added later on, so the list might end up large.

Reply:
👍 Lets do it once we get there 😉

Reply:
Sure 👍